File: Uri/1038.c
#include<stdio.h>
#include<stdlib.h>

int main(){
    int x, q;
    double t;
    scanf("%d %d", &x, &q);
    if(x == 1){
        t = q * 4.00;
    }
    else if(x == 2){
        t = q * 4.50;
    }
    else if(x == 3){
        t = q * 5.00;
    }
    else if(x == 4){
        t = q * 2.00;
    }
    else{
        t = q * 1.50;
    }
    printf("Total: R$ %.2lf\n", t);
    return 0;
}

File: Uri/1038.py
x = raw_input();
x = x.split(' ');
a = int(x[0])
b = int(x[1])

if a == 1:
    i = 4.0*b
    print "Total: R$ %.2f"%i
elif a == 2:
    i = 4.5*b
    print "Total: R$ %.2f"%i
elif a == 3:
    i = 5.0*b
    print "Total: R$ %.2f"%i
elif a == 4:
    i = 2.0*b
    print "Total: R$ %.2f"%i
elif a == 5:
    i = 1.5*b
    print "Total: R$ %.2f"%i
